### PR TITLE
feat: Distributed plugin Dispatcher

### PR DESCRIPTION
**Pull request by dovics**
Signed-off-by: dovics <wrs369@163.com>

### CODE REVIEW

File: persist/dispatcher.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 The casbin Authors. All Rights Reserved.
```
Comment (nodece) [inline persist/dispatcher.go:1]: update casbin to Casbin.

Comment (dovics) [reply persist/dispatcher.go]: fixed, but all other files are casbin

Comment (nodece) [reply persist/dispatcher.go]: sorry, I don't note for other files.

**Comment by nodece** [comment]
@dovics could you help me squash the commit? I made a change merge the PR into beta branch.

**Pull request merged**